### ISSUE TITLE
Need help 

### ISSUE THREAD

**Issue by luanruby** [abhn/Elementary-Wordpress#24]
when installing the theme for Wordpress reported an error that the file could not be found:  style.css
help me update this error. thank

**Comment by abhn** [comment]
Sorry, I've not updated the readme yet. Will do soon.

You need to first run the following commands
```
npm install
npm run compile:css
npm run compile:js
dir-archiver --src . --dest ../_s.zip --exclude .DS_Store .stylelintrc.json .eslintrc .git .gitattributes .github .gitignore README.md composer.json composer.lock node_modules vendor package-lock.json package.json .travis.yml phpcs.xml.dist sass style.css.map
```

and then upload the `_s.zip` file generated to your wordpress website

**Comment by luanruby** [comment]
Thanks for answering. but i'm not knowledgeable about code, i wait for the
latest update.

Vào Th 5, 31 thg 3, 2022 vào lúc 14:25 Abhishek Nagekar <
***@***.***> đã viết:

**Comment by devdil** [comment]
@abhn what node version are you using?

**Comment by abhn** [comment]
I don't recall that (my bad, should've documented). Is it not building?

**Comment by devdil** [comment]
No, it isn’t. Could you provide a docker image for the environment?

**Comment by devdil** [comment]
@abhn do you have a docker image?

**Comment by abhn** [comment]
No, sorry. It is on my todo, but I'm yet to get to it. @devdil
